tsconfig.json

```diff
@@ -6,7 +6,9 @@
     "rootDir": "./src",
     "strict": true,
     "noImplicitAny": true,
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "sourceMap": true,
+    "inlineSourceMap": false
```

Comment:
Setting 'inlineSourceMap' to false while 'sourceMap' is true means that source maps will be generated as separate files. Ensure this aligns with your project's debugging strategy and requirements.